Student_BCOs/(FinalUpdate)Juntian_Wei_BCO_000549.json

```diff
@@ -0,0 +1,434 @@
+                    "email": "janishapatel@gwu.edu"
+                },
+                "date": "10_16_2019"
+            }
+        ],
+        "contributors": [
+            {
+                "name": "Juntian Wei",
+                "orcid": "https://orcid.org/0000-0002-5378-1520",
+                "affiliation": "George Washington University",
+                "contribution": [
+                    "createdBy"
+                ],
+                "email": "jwei48@gwmail.gwu.edu"
+            },
+            {
+                "name": "Giorgio Giurato",
+                "orcid": "https://orcid.org/0000-0002-0538-8978",
+                "affiliation": "Laboratory of Molecular Medicine and Genomics, Department of Medicine and Surgery, University of Salerno, via Allende, 1, Salerno, Baronissi, Italy",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Maria Rosaria De Filippo",
+                "orcid": "https://orcid.org/0000-0001-8350-2599",
+                "affiliation": "Fondazione IRCCS SDN, Napoli, Italy",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Antonio Rinaldi",
+                "orcid": "https://orcid.org/0000-0001-6777-9512",
+                "affiliation": "Laboratory of Molecular Medicine and Genomics, Department of Medicine and Surgery, University of Salerno, via Allende, 1, Salerno, Baronissi, Italy",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Adnan Hashim",
+                "orcid": "https://orcid.org/0000-0003-4736-8315",
+                "affiliation": "Laboratory of Molecular Medicine and Genomics, Department of Medicine and Surgery, University of Salerno, via Allende, 1, Salerno, Baronissi, Italy",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Giovanni Nassa",
+                "orcid": "https://orcid.org/0000-0001-7453-1240",
+                "affiliation": "Laboratory of Molecular Medicine and Genomics, Department of Medicine and Surgery, University of Salerno, via Allende, 1, Salerno, Baronissi, Italy",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Maria Ravo",
+                "orcid": "https://orcid.org/0000-0003-1003-2922",
+                "affiliation": "Laboratory of Molecular Medicine and Genomics, Department of Medicine and Surgery, University of Salerno, via Allende, 1, Salerno, Baronissi, Italy",
+                "contribution": [
+                    "authoredBy"
```

Comment:
In addition to "authoredBy", each publication author should also have the "derivedFrom" contribution type. Please add. See #113